client/src/main/java/org/apache/rocketmq/client/producer/DefaultMQProducer.java

```diff
@@ -72,7 +72,8 @@ public class DefaultMQProducer extends ClientConfig implements MQProducer {
         ResponseCode.SYSTEM_ERROR,
         ResponseCode.NO_PERMISSION,
         ResponseCode.NO_BUYER_ID,
-        ResponseCode.NOT_IN_CURRENT_UNIT
+        ResponseCode.NOT_IN_CURRENT_UNIT,
+        ResponseCode.SYSTEM_BUSY
```

Comment:
How about removing TOPIC_NOT_EXIST?

Reply:
We don't know the intent of the author who wrote it, so removing it is probably not a good option. Although I also think that client should not retry when broker returns `TOPIC_NOT_EXIST`:smile:. @ShadowySpirits

Reply:
@wz2cool  WDYT? :-)

Reply:
System busy means server is overload, why we need retry send message?

Reply:
> System busy means server is overload, why we need retry send message?

To make sure message which send failed can retry to send success to another brokers.

Reply:
we have  some disussion about it  plz see #2726

Reply:
> How about removing TOPIC_NOT_EXIST?
 
Sometimes a topic will be deleted or disabled on a broker node. Maybe due to the failed routing update, it may request this broker, so retrying here is still necessary. @ShadowySpirits